main.tex

```diff
@@ -576,10 +542,10 @@ \section{Parameter Error Estimation for Ammonia}
 \caption{Ammonia model profile fit to a synthetic spectrum.
 The parameter values and errors are shown in the upper right.  The associated
 error estimate triangle diagram is shown in Figure \ref{fig:nh3parerrestdemo}.
-The correct parameters are $T_{R}=T_{ex}=35$, $N=15$, $\sigma=2$, and $v=0$,
-all of which are reasonably recovered.  However, note that $T_{ex} > T_{rot}$
-is generally nonphysical, yet the allowed parameter space for $T_{ex}$ includes
-such values.
+The correct parameters are $T_{\mathrm{R}}=T_{\mathrm{ex}}=35$, $N=15$, $\sigma_v=2$, and $v=0$,
+all of which are reasonably recovered.  However, note that $T_{\mathrm{ex}} > T_{\mathrm{R}}$
+is generally nonphysical, yet the allowed parameter space for $T_{\mathrm{ex}}$ includes
+such values. \textcolor{red}{EWR: Not showing the empty spectral region would make a nicer visualization, IMHO.}
```

Comment:
You mean the space between the 1-1 and 2-2 lines?  It's just inconvenient to code that up, and I do need to show both... I guess I could use the wrapper and make a two-panel figure...